ds0_utils.py

```diff
@@ -75,3 +81,38 @@ def required_commit():
 
     if options and options.forcecommit:
         return options.forcecommit
+
+def repository():
+    options = parse_ec2_userdata()
+
+    if options and options.repository:
+        if ':' in options.repository:
+            (origin, branch) = options.repository.split(':')
+        else:
+            origin = options.repository
+            branch = ''
+        return { 'origin': origin, 'branch': branch }
+    else:
+        return { 'origin': '', 'branch': '' }
+
+def disable_commit_verification():
+    options = parse_ec2_userdata()
+
+    if options and options.disablecommitverification:
+        return options.disablecommitverification
+    else:
+        return False
+
+def allowed_keys():
+    options = parse_ec2_userdata()
+
+    if options and options.allowedkeys:
+        return options.allowedkeys
+    else:
+        default_key = {
+            'id': '7123CDFD',
+            'file': '/home/ubuntu/datastax_ami/repo_keys/DataStax_AMI.7123CDFD.key',
```

Comment:
First, clarification - when I said "feel free to merge it as-is" I was referring to the key file naming convention, not the hard-coded path. :)

Since the primary consumer of the key files is a program, I think it makes sense for the naming convention to be easily processed with one. I'll submit a separate pull request that addresses this issue once we're happy with these changes.